### ISSUE TITLE
[DEV branch] cannot run runserver.py

### ISSUE THREAD

**Issue by daproject85** [AHAAAAAAA/PokemonGo-Map#2111]
/runserver.py -a google -u XXXXX -p XXXXX -l "<LOCATION>" -st 10 -k "XXXXXXX" -t 2 -c false
Traceback (most recent call last):
  **File "./runserver.py", line 10, in <module>
    from flask_cors import CORS, cross_origin
ImportError: No module named flask_cors**

**Comment by reyqn** [comment]
pip install -r requirements.txt maybe?

**Comment by daproject85** [comment]
done it all

**Comment by reyqn** [comment]
Well no idea then. That's what it seems to mean

**Comment by theblackmarmoset** [comment]
Did you install the most recent dev branch?

**Comment by RemyLebeau** [comment]
pip is currently giving me time out errors ... connection refused

**Comment by daproject85** [comment]
yea @theblackmarmoset literally just now

**Comment by daproject85** [comment]
i have flask_cors too, 


ImportError: No module named flask_cors
:PokemonGo-Map-develop Rocky$ pip install flask_cors
Requirement already satisfied (use --upgrade to upgrade): flask_cors in /usr/local/lib/python2.7/site-packages
Requirement already satisfied (use --upgrade to upgrade): Six in /usr/local/lib/python2.7/site-packages (from flask_cors)
Requirement already satisfied (use --upgrade to upgrade): Flask>=0.9 in /usr/local/lib/python2.7/site-packages (from flask_cors)
Requirement already satisfied (use --upgrade to upgrade): click>=2.0 in /usr/local/lib/python2.7/site-packages (from Flask>=0.9->flask_cors)
Requirement already satisfied (use --upgrade to upgrade): Jinja2>=2.4 in /usr/local/lib/python2.7/site-packages (from Flask>=0.9->flask_cors)
Requirement already satisfied (use --upgrade to upgrade): Werkzeug>=0.7 in /usr/local/lib/python2.7/site-packages (from Flask>=0.9->flask_cors)
Requirement already satisfied (use --upgrade to upgrade): itsdangerous>=0.21 in /usr/local/lib/python2.7/site-packages (from Flask>=0.9->flask_cors)
Requirement already satisfied (use --upgrade to upgrade): MarkupSafe in /usr/local/lib/python2.7/site-packages (from Jinja2>=2.4->Flask>=0.9->flask_cors)
:PokemonGo-Map-develop Rocky$ pip install flask_cors

**Comment by theblackmarmoset** [comment]
Okay.  So I'm guessing you haven't run `npm install` or `npm install grunt` yet?

**Comment by daproject85** [comment]
...no... we never needed to do that, why need it now? @theblackmarmoset

**Comment by daproject85** [comment]
and what do those commands dO?

**Comment by theblackmarmoset** [comment]
Yes we need those commands now.   Open a command window in the folder and run those commands.

**Comment by daproject85** [comment]
ok just did, canyou explain what they do ? and hwy we need them now? and also ... i looked in my python library, flask_cors is there but there are no CORS or cross_orgin

**Comment by theblackmarmoset** [comment]
Also you need to run `npm install -g grunt grunt jshint sass cssmin uglify` as well otherwise it won't show the map because there are some missing files in grunt.  Those commands will install grunt.  That's all I know.  The wiki was updated so I really haven't had time to take a look to see what they do.

**Comment by theblackmarmoset** [comment]
And maybe try running `pip uninstall requirements.txt`  and then running `pip install -r requirements.txt`

**Comment by daproject85** [comment]
@theblackmarmoset wiki just sais npm install ... tahts it ....

**Comment by theblackmarmoset** [comment]
Yeah, but with just that it won't work.

**Comment by scooterpsu** [comment]
The wiki instructions work just fine with the **master** branch. If you want to use the latest _develop_ code, there's more involved.

**Comment by theblackmarmoset** [comment]
Yeah definitely.  Most of the time I just post my problems on here and kind strangers are able to help me out!  It's also helped me learn a lil bit about programming too.

**Comment by daproject85** [comment]
i agree.... also there is no more credentials.json? you have to enter your google map apis on CL?

**Comment by theblackmarmoset** [comment]
Yes.  And make sure you use the protocol -k [API key] when running the server.  Otherwise it will not work.

**Comment by daproject85** [comment]
@theblackmarmoset yea ... cuz i put the api key in the config.ini, but it wont take it

**Comment by theblackmarmoset** [comment]
you have to put the api key in the config.ini and the command you use to run the server using -k [API key].

**Comment by daproject85** [comment]
so did all that... map dont load .......... 


<img width="1475" alt="screen shot 2016-07-25 at 4 49 21 pm" src="https://cloud.githubusercontent.com/assets/2730554/17121330/c62f2014-5287-11e6-8336-04b76bf28876.png">

**Comment by bsphil** [comment]
That error was just rolled back in the most recent commit a couple minutes ago.  Try pulling updates to the develop branch and restarting the server.

**Comment by daproject85** [comment]
which error? @bsphil

**Comment by willgrnr** [comment]
@daproject85 this entire issue was fixed in a recent revert commit.

**Comment by daproject85** [comment]
@willgrnr my friend i dont see any commits, i got this little over an hour ago... dont see anything about this ... but which issue? the ones about the npm and grunt modules needing to be manually installed?

**Comment by willgrnr** [comment]
#2114 fixes the issue with having to manually install and a lack of stylesheet.

**Comment by Zach1993** [comment]
@daproject85 are you using github desktop to pull the newest commits? Or are you just downloading the zip file?

**Comment by daproject85** [comment]
manually downloading zipfile.

**Comment by wowfood** [comment]
Having the same issue, just downloaded the master branch.

Installed python 27
Installed pip
running pip install -r requirements.txt seems to fail.  Errors appearing are

Failed building wheel for MarkupSafe

and

Command "c:\python27\python.exe -u -c "import setuptools, tokenize;__file__='c:\\users\\aidene~1\\appdata\\local\\temp\\pip-build-raumyj\\MarkupSafe\\setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record c:\users\aidene~1\appdata\local\temp\pip-irkugj-record\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in c:\users\aidene~1\appdata\local\temp\pip-build-raumyj\MarkupSafe\

npm install worked fine however.

**Comment by Amdaxiom** [comment]
Thank you wowfood, that solved my issue
pip install setuptools==21.2.1

then re-running pip install -r requirements.txt
